controllers/applicationvolumereplication_controller.go

```diff
@@ -188,105 +206,330 @@ func getMostRecentConditions(conditions []metav1.Condition) []metav1.Condition {
+	// cluster, unpause the subscription, and skip it until the next reconciler iteration
+	if r.isSubsriptionPausedForDR(subscription.GetLabels()) {
+		newHomeCluster, err := r.processPausedSubscription(avr, subscription)
+		if err != nil {
+			r.Log.Error(err, fmt.Sprintf("failed to process paused Subscription (%v)", subscription))
 
-	avr.Status = ramendrv1alpha1.ApplicationVolumeReplicationStatus{
-		Decisions: placementDecisions,
+			return nil, requeue
+		}
+
+		r.Log.Info(fmt.Sprintf("Unpausing subscription %s for new home cluster %s", subscription.Name, newHomeCluster))
+
+		err = r.unpauseSubscription(subscription)
+		if err != nil {
+			r.Log.Error(err, "failed to unpause subscription", "name", subscription.Name)
+
+			return nil, requeue
+		}
+
+		// Subscription has been unpaused. Stop processing it and wait for the next Reconciler iteration
+		r.Log.Info("Subscription unpaused. It will be processed in the next reconciler iteration", "name", subscription.Name)
+
+		return nil, requeue
 	}
-	if err := r.Client.Status().Update(ctx, avr); err != nil {
-		return errorswrapper.Wrap(err, "failed to update AVR status")
+
+	exists, err := r.vrgManifestWorkAlreadyExists(avr, subscription)
+	if err != nil {
+		return nil, requeue
+	}
+
+	if exists {
+		return nil, !requeue
+	}
+	// This subscription is ready for manifest (VRG) creation
+	placementDecision, err := r.processUnpausedSubscription(avr, subscription)
+	if err != nil {
+		r.Log.Error(err, "Failed to process unpaused subscription", "name", subscription.Name)
+
+		return nil, requeue
+	}
+
+	r.Log.Info(fmt.Sprintf("placementDecisions %v - requeue: %t", placementDecision, !requeue))
+
+	return &placementDecision, !requeue
+}
+
+func (r *ApplicationVolumeReplicationReconciler) isSubsriptionPausedForDR(labels map[string]string) bool {
+	return labels != nil &&
+		labels[RamenDRLabelName] != "" &&
+		strings.EqualFold(labels[RamenDRLabelName], "protected") &&
+		labels[subv1.LabelSubscriptionPause] != "" &&
+		strings.EqualFold(labels[subv1.LabelSubscriptionPause], "true")
+}
+
+// processPausedSubscription selects the target cluster from the subscription or
+// from the user selected cluster and restores all PVs (belonging to the subscription)
+// to the target cluster and then it unpauses the subscription
+func (r *ApplicationVolumeReplicationReconciler) processPausedSubscription(
+	avr *ramendrv1alpha1.ApplicationVolumeReplication,
+	subscription *subv1.Subscription) (string, error) {
+	r.Log.Info("Processing paused subscription", "name", subscription.Name)
+
+	// find new home cluster (could be the failover cluster)
+	newHomeCluster := r.findNextHomeCluster(avr, subscription)
+
+	if newHomeCluster == "" {
+		return "", fmt.Errorf("failed to find new home cluster: avr %s, subscription %s", avr.Name, subscription.Name)
+	}
+
+	mw, err := r.findManifestWork(subscription, newHomeCluster, MWTypePV)
+	if err != nil {
```

Comment:
Deleting the PV MW is only needed in the failback case.  SO it will come later.
But, you made a good point about the latest PVs.  If the S3 interface wrapper will not handle it, then we should talk.
cc: @vdeenadh

Reply:
When an app requests a PVC & thus a PV, VRG will detect it and do the following: 
(1) upload PV metadata to S3 and proceed to next step only when the upload is successful 
(2) create VR to replicate data.  

If a PV didn't finish step (1), then such a PV's data is also not replicated to the peer cluster.  Every such unreplicated PV will thereby result in a new PV to be created upon failback to fulfill the PVC.  Cleanup of such unreplicated lost PVs will be handled similar to cleanup of PVs when the app is deleted.